dolphin/processor/config.py

```diff
@@ -81,7 +81,10 @@ def pixel_size(self):
         :return:
         :rtype:
         """
-        return self.settings['pixel_size']
+        if type(self.settings['pixel_size']) == float:
```

Comment:
What would happen if a user provides multiple bands, but just one float value for the pixel size because it's same for all the bands?

Reply:
Sorry my mistake. It used to work when I had pixel_size  returns either float and list, but I forgot updated it when  I changed pixel_size  so that it only return a list